source/_docs/terminus/scripting.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Terminus Manual
+subtitle: Scripting Terminus
+description: Automate your workflow with Terminus.
+terminuspage: true
+type: terminuspage
+layout: terminuspage
+nexturl: terminus/plugins/
+previousurl: terminus/commands/
+permalink: docs/terminus/:basename/
+image: terminus-thumbLarge
+searchboost: 100
+---
+
+While Terminus makes it easy to manually manipulate the Pantheon environment from your command line, its real value is in scripting. By adding Terminus to your automated tasks, you can truly make Pantheon a part of your standardized workflow.
+
+Consider the repetitive tasks you perform using the Pantheon Dashboard:
+
+ - Can those tasks be executed by Terminus commands?
+ - Can the values required by the commands be derived programmatically?
+
+If so, you consider how you can turn the task into a script.
+
+## Authentication
+
+To streamline your use of Terminus in scripts, make sure that it's authenticated first. The recommended method is to use a [machine token](/docs/terminus/install#machine-token).
```

Comment:
> To streamline your use of Terminus in scripts, make sure that it's authenticated first. 

Maybe something like "Terminus needs to be authenticated in order to execute most commands. Before running any script you must ensure Terminus is authenticated with a machine token that has the proper permissions"